frontend/lib/rental-history.tsx

```diff
@@ -29,13 +30,25 @@ function RentalHistoryWelcome(): JSX.Element {
   }
 
 function RentalHistoryForm(): JSX.Element {
+  const appContext = useContext(AppContext);
+  const userData = appContext.session;
+
+  const UserRhFormInput: RhFormInput = (userData && userData.userId ?
+    {
+      "firstName": ( userData.firstName || "" ),
+      "lastName": ( userData.lastName || "" ),
+      "address": ( (userData.onboardingInfo && userData.onboardingInfo.address) || "" ) + ", " + ( (userData.onboardingInfo && userData.onboardingInfo.borough) || "" ),
```

Comment:
Ah, so you'll actually be wanting to have `borough` be a separate field from address ultimately.  You can do that in this PR or save it for another one.